src/prelude.js

```diff
@@ -352,9 +357,17 @@ internals.symbolValue = function (symbol){
   }
 };
 
+internals.fboundp = function (symbol) {
+  if (symbol instanceof internals.Symbol){
+    return !symbol.fvalue[UNBOUND]
+  } else {
+    throw new Error(`${symbol} is not a symbol`)
```

Comment:
Function code can be large. 
Can we shorten this message?
As an example:
ERROR: FBOUNDP - Invalid function name `#<FUNCTION CAR>`.

Or can wouldbe use a wrapper in bootstrap.lisp:
```lisp
(defun fboundp (x)
  (if (functionp x)
      (error "FBOUNDP - invalid function name ~a." x))
  (fboundp x))
```
```lisp
CL-USER> (fboundp #'car)
ERROR: FBOUNDP: invalid function name #<FUNCTION CAR>.
CL-USER>
```

Reply:
It would be cool to expose those things in a structured way and with conditions indeed.

Reply:
I guess `fboundp` is not super important for performance, so I'll do that :-)

Reply:
>However the compiler prefers the inline form, unless we change the name of the primtivie to %fboundp, it would still emit code for it inline instead of a function call!

A crutch that has stood the test of time :-)
```lisp
;;; file bootstrap.lisp
(defun fboundp (x)
  (if (functionp x)
      (error "FBOUNDP - invalid function name ~a." x))
  (fboundp x))
```